bondcpp/include/bondcpp/bond.hpp

```diff
@@ -58,6 +57,30 @@ namespace bond
  */
 class Bond
 {
+public:
```

Comment:
Nit: stylistically, I'm not a huge fan of multiple public/private declarations in a class.  I'd rather these be moved into the respective heading below.